### PR TITLE
[GEOS-5090] Show styles to be deleted in GUI notification

### PR DESCRIPTION
**Pull request by tbarsballe**
https://osgeo-org.atlassian.net/browse/GEOS-5090

### CODE REVIEW

**Comment by jodygarnett** [comment]
Quesiton - can this be backported as a bug fix? Or is it a "new" feature?

File: src/web/core/src/test/java/org/geoserver/web/data/ConfirmRemovalPanelTest.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
+/* (c) 2014 - 2016 Open Source Geospatial Foundation - all rights reserved
```
Comment (jodygarnett) [inline src/web/core/src/test/java/org/geoserver/web/data/ConfirmRemovalPanelTest.java]: No need to update the header anymore :)

**Comment by jodygarnett** [comment]
Looks good!

File: src/web/core/src/test/java/org/geoserver/web/data/ConfirmRemovalPanelTest.java (hunk 1 above)
Comment (tbarsballe) [inline src/web/core/src/test/java/org/geoserver/web/data/ConfirmRemovalPanelTest.java]: Is that now official? Excellent. Has the Contributing guide been updated?

**Comment by tbarsballe** [comment]
This can be backported as far back as we allow workspaced styles, so yes. I will do so after merging.

**Pull request merged**